gobblin-utility/src/main/java/gobblin/util/JobLauncherUtils.java

```diff
@@ -179,19 +183,30 @@ public static void cleanStagingData(State state, Logger logger, Closer closer,
     }
   }
 
-  private static FileSystem getFsWithProxy(State state, String writerFsUri) throws IOException {
-    if (state.getPropAsBoolean(ConfigurationKeys.SHOULD_FS_PROXY_AS_USER,
+  private static FileSystem getFsWithProxy(State state, String writerFsUri)
+      throws IOException {
+    if (!state.getPropAsBoolean(ConfigurationKeys.SHOULD_FS_PROXY_AS_USER,
         ConfigurationKeys.DEFAULT_SHOULD_FS_PROXY_AS_USER)) {
-      try {
-        return new ProxiedFileSystemWrapper().getProxiedFileSystem(state, ProxiedFileSystemWrapper.AuthType.KEYTAB,
-            state.getProp(ConfigurationKeys.SUPER_USER_KEY_TAB_LOCATION), writerFsUri);
-      } catch (InterruptedException e) {
-        throw new IOException(e);
-      } catch (URISyntaxException e) {
-        throw new IOException(e);
-      }
-    } else {
       return FileSystem.get(URI.create(writerFsUri), new Configuration());
+    } else {
+      Preconditions.checkArgument(StringUtils.isNotBlank(state.getProp(ConfigurationKeys.FS_PROXY_AS_USER_NAME)),
```

Comment:
i think the standard in gobblin is to use Guava's `Strings.isNullOrEmpty` method.